File: challenges/2-idade-em-dias/python/eric-vantunes/idade-em-dias.py
idade = int(input())

anos = int(idade/365)
saldo = idade - (anos * 365)

meses = int(saldo/30)
dias = saldo - meses * 30

print(anos, "ano(s)")
print(meses, "mes(es)")
print(dias, "dia(s)")

File: challenges/2-idade-em-dias/javascript/barbarabrito/idadeEmDias.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
  });

readline.question('Idade em dias: ', (idadeEmDias) => {

    let anos =  Math.floor(idadeEmDias / 365);
    let meses =  Math.floor((idadeEmDias % 365) / 30);
    let dias =  Math.floor((idadeEmDias % 365) % 30);
    
    console.log(`\n ${anos} ano(s) \n ${meses} mes(es) \n ${dias} dia(s) \n`)
    readline.close();

  });